metrics-server/manifest.yaml

```diff
@@ -0,0 +1,9 @@
+---
+name: metrics-server
```

Comment:
Please could you name this as Metrics Server, the name is used as a title in display on the site.

Reply:
Sure